### PR TITLE
T-21498 - Make nft.burns Trino compatible

### PR DESCRIPTION
**Pull request by belen-pruvost**

### CODE REVIEW

**Comment by belen-pruvost** [comment]
I've run all the models locally successfully.

File: models/zora/ethereum/zora_ethereum_events.sql [hunk 1]
```diff
@@ -328,7 +328,7 @@ SELECT distinct 'ethereum' AS blockchain
     , zt.buyer
     , CASE WHEN zt.currency_contract='0x0000000000000000000000000000000000000000' THEN zt.amount_raw/POWER(10, 18)
         ELSE zt.amount_raw/POWER(10, pu.decimals) END AS amount_original
-    , zt.amount_raw
+    , CAST(zt.amount_raw AS DECIMAL(38,0)) AS amount_raw
```
Comment (jackromo888) [inline models/zora/ethereum/zora_ethereum_events.sql:331]:
```suggestion
    , CAST(zt.amount_raw AS - [ ] - [ ] DECIMAL(38,0)) AS amount_rawhttps://github.com/duneanalytics/spellbook/blob/1bb57692dd41ab39e0397afab403e71828598577/models/blur/ethereum/blur_ethereum_events.sql
```

**Review by jackromo888 — APPROVED** [review]

**Review by soispoke — APPROVED** [review]

**Comment by belen-pruvost** [comment]
Let's hold on this one, just in case (added WIP label)

**Comment by belen-pruvost** [comment]
I'll close this one to follow the process outlined in the postmortem doc later.

**Pull request closed without merging**